WMFPageHistorySection.h

```diff
@@ -0,0 +1,17 @@
+//
+//  WMFPageHistorySection.h
+//  Wikipedia
+//
+//  Created by Nick DiStefano on 4/3/16.
+//  Copyright © 2016 Wikimedia Foundation. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+@class WMFPageHistoryRevision;
+
+@interface WMFPageHistorySection : NSObject
+
+@property (nonatomic, copy, readwrite) NSString* _Nullable sectionTitle;
```

Comment:
@coreyfloyd 

on second thought I see a simple way to make this immutable and keep the current parsing logic. It would involve a good deal of object creation which I'm inclined to shy away from when parsing large data sets, but since there are only 50 items I guess it's ok